nodes/technic_chests.lua

```diff
@@ -53,22 +53,35 @@ local function with_owner_field(metas)
 	return result
 end
 
+local function description(pos, meta, node, player)
+	local infotext = meta:get_string('infotext')
+	-- Default label starts with an escape char for translated text.
+	if 27 == infotext:sub(1, 1):byte() then
+		return wrench.description_with_items(pos, meta, node, player)
+	end
+
+	return infotext
```

Comment:
Not sure if the formspec escape is needed for item descriptions, but better to be safe.
```suggestion
	if infotext:sub(1, 1):byte() == 27 then
		return wrench.description_with_items(pos, meta, node, player)
	end
	return minetest.formspec_escape(infotext)
```

Reply:
yeah, I was considering that.

Reply:
You didn't fix the Yoda condition.